### PR TITLE
fix: make the example run correctly

### PR DESCRIPTION
**Pull request by ou-bing**
**What this PR does**:

make the example run correctly

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ services:
     networks:
       - local
     ports:
-      - "3307:13306"
+      - "13306:13306"
```
[the remark this answers is not in the document]
Comment (ou-bing) [reply docker-compose.yaml:72]: Or we can modify the configuration file of the demo ？
We must ensure that the demo works properly.

[the remark this answers is not in the document]
Comment (jjeffcaii) [reply docker-compose.yaml:72]: Which demo?

[the remark this answers is not in the document]
Comment (ou-bing) [reply docker-compose.yaml:72]: I refer to this document：https://arana.yuque.com/staff-guqen5/pto7y5/ye22qw。

Run the `docker compose up` command under the project root directory to start the demo

[the remark this answers is not in the document]
Comment (jjeffcaii) [reply docker-compose.yaml:72]: For demo, use `docker compose up` to quick start an arana server, and you can connect over port 3307.
Bug for arana developer, you need also run `example/local_server` to start a debug server which listen on 13306, it won't conflict with port 3307 (docker container).

**Pull request closed without merging**